modules/vpc-sc/main.tf

```diff
@@ -48,9 +48,12 @@ resource "google_access_context_manager_access_level" "default" {
     content {
       combining_function = try(each.value.combining_function, null)
       conditions {
-        ip_subnetworks = try(basic.value.ip_subnetworks, null)
-        members        = try(basic.value.members, null)
-        negate         = try(basic.value.negate, null)
```

Comment:
Nit: can you add `iterator = condition` and then use `condition.value` instead of `basic.value`? It would probably increase readability.

Reply:
done